kogito-kafka-quickstart-quarkus/README.md

```diff
@@ -82,6 +154,11 @@ To make use of this application it is as simple as putting a message on `travell
 	}
 }
 ```
+One liner
+```
+{"specversion": "0.3","id": "21627e26-31eb-43e7-8343-92a696fd96b1","source": "","type": "VisaApplicationsMessageDataEvent_8", "time": "2019-10-01T12:02:23.812262+02:00[Europe/Warsaw]","data": { "firstName" : "Jan", "lastName" : "Kowalski", "email" : "jan.kowalski@example.com", "nationality" : "Polish"}}
```

Comment:
same here, should be `TravellersMessageDataEvent_3`

Reply:
fixed @radtriste